flit/inifile.py

```diff
@@ -117,7 +120,14 @@ def read_pkg_ini(path):
     if 'description-file' in md_sect:
         description_file = path.parent / md_sect.pop('description-file')
         with description_file.open() as f:
-            md_dict['description'] = f.read()
+            raw_desc =  f.read()
+        stream = io.StringIO()
+        _, ok = render(raw_desc, stream)
+        stream.seek(0)
+        if not ok:
+            log.warn("The file description seem to not be valid rst for PyPI it will be interpreted as plain text")
```

Comment:
* seem -> seems
* to not be -> not to be
* some punctuation after PyPI - I'd use a semicolon